Makefile

```diff
@@ -55,20 +57,26 @@ clean-vendor:
 	sh script/clean_vendor.sh
 
 .PHONY: check
-check:
+check: python-env
 	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) check || exit 1;)
+	# Checks also python files which are not part of the beats
+	find . -type f -name *.py -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*" -exec autopep8 -d --max-line-length 120  {} \; | (! grep . -q) || (echo "Code differs from autopep8's style" && false)
 	# Validate that all updates were committed
 	$(MAKE) update
 	git update-index --refresh
 	git diff-index --exit-code HEAD --
 
+# Corrects spelling errors
+.PHONY: misspell
+misspell:
+	go get github.com/client9/misspell
+	find . -type f -name '*' -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*" -exec misspell -w {} \;
```

Comment:
since this `find` command is used in quite a few places, it might make sense to alias parts of it. Something like `FIND=find -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*"`